appium/tests/specs/composeEmail/SendEmailToRecipientWithRevokedPublicKey.spec.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  SplashScreen,
+  SetupKeyScreen,
+  MailFolderScreen,
+  NewMessageScreen
+} from '../../screenobjects/all-screens';
+
+import { CommonData } from '../../data';
+
+describe('COMPOSE EMAIL: ', () => {
+
+  it('sending message to user with revoked public key produces modal', async () => {
```

Comment:
All good, just please make both tests in a single spec. The setup / authentication / etc takes a long time, so if the tests are simple and related, it's better to put them together to save developer's time waiting for tests to finish each time. Thanks!

Reply:
ok, will change